Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+
+[requires]
+python_version = "3.8"
```

Comment:
Should this be 3.7?

Reply:
Probably yes, I was actually wondering if we even need the Pipfile at the root - I'll change it, but the file might also disappear :)